Readme.md

```diff
@@ -0,0 +1,31 @@
+# browser-support-checker
+Lightweight browser support checker
+
+## Install
+
+```
+git clone https://github.com/vanya6537/browser-checker-tao
+cd browser-checker-tao 
+npm install
```

Comment:
`npm i --save @oat-sa/browser-support-checker`

and can you also quickly document how to use it.